cirq/linalg/decompositions_test.py

```diff
@@ -240,6 +240,18 @@ def test_kak_decomposition(target):
     np.testing.assert_allclose(cirq.unitary(kak), target, atol=1e-8)
 
 
+def test_kak_decomposition_interaction_coefficients():
+    a, b = cirq.LineQubit(0), cirq.LineQubit(1)
+    theta = 0.3
+    circuit = cirq.Circuit()
+    circuit.append(cirq.ISWAP.on(a, b))
+    circuit.append(
+        cirq.ZZPowGate(exponent=2 * theta / np.pi, global_shift=-0.5).on(a, b))
+
+    assert cirq.kak_decomposition(cirq.unitary(circuit)).interaction_coefficients == \
```

Comment:
Don't use exact equality with floating point numbers.

Reply:
I have removed this because it was an attempt to reproduce the kak bug now that we know the reason we were getting it.